app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -371,6 +371,7 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                 val txDetailsRejectionBinding = contentBinding as TxDetailsRejectionBinding
                 with(txDetailsRejectionBinding) {
                     txRejectionInfo.text = getString(R.string.tx_details_rejection_info, nonce)
+                    txPaymentReasonLink.text = null
```

Comment:
Sorry. I should have warned you about this append behaviour. Should we have a warning about this as a comment?

Reply:
check out new set link extension